config/routes.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  # get 'root/index'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  root "message#index"
+end
```

Comment:
- [ ] It appears that you have a syntax error leading to an API error. In your controllers, you have MessagesController with index. So, the route is supposed to be `messages#index`